website/src/Pages/ResourcesPage/AddResourcePage.jsx

```diff
@@ -0,0 +1,243 @@
+  const [endTime, setEndTime] = useState(dayjs().add(1, 'h'));
+  const [latitude, setLatitude] = useState('');
+  const [longitude, setLongitude] = useState('');
+  const [address, setAddress] = useState('');
+  const [email, setEmail] = useState('');
+  const [website, setWebsite] = useState('');
+
+  const updateDays = (num) => {
+    const updatedDaysStatus = [...days];
+    updatedDaysStatus[num] = updatedDaysStatus[num] ? 0 : 1;
+    setDays(updatedDaysStatus);
+  };
+
+  const backToResources = () => {
+    navigate('/resources');
+  };
+
+  const handlePhoneNumberChange = (event) => {
+    const inputValue = event.target.value;
+
+    // Remove all non-numeric characters from the input value
+    const numericValue = inputValue.replace(/\D/g, '');
+
+    // Format the phone number once it reaches 10 digits
+    if (numericValue.length === 10) {
+      const formattedValue = `(${numericValue.slice(0, 3)}) ${numericValue.slice(3, 6)}-${numericValue.slice(6, 10)}`;
+      setPhoneNumber(formattedValue);
+    } else if (numericValue.length < 10) {
+      setPhoneNumber(numericValue);
+    }
+  };
+
+  const createResource = async () => {
+    const daysOfWeek = [];
+    days.forEach((day, index) => {
+      if (day) {
+        daysOfWeek.push(index);
+      }
+    });
+    const dateDetails = {
+      days: daysOfWeek,
+      startTime: startTime.format('h:mm a'),
+      endTime: endTime.format('h:mm a'),
+    };
+    const locationDetails = {
+      address,
+      coordinates: {
+        latitude: Number(latitude),
+        longitude: Number(longitude),
+      },
+    };
+    const newResource = {
+      title,
+      dateDetails,
+      locationDetails,
+      tag,
+      phoneNumber,
+      email,
+      website,
+    };
+    await axios.post('http://localhost:4000/resource/post', newResource, {
+      headers: authHeader,
+    });
+    backToResources();
+  };
+
+  return (
+    <Container>
+      <Box>
+        <Typography variant="h5">Add Resource Card</Typography>
+      </Box>
+      <Box>
+        <Typography>Resource Title</Typography>
+        <TextField
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+        />
+      </Box>
+      <Box>
+        <Typography>Tag</Typography>
+        <FormControl fullWidth>
+          <Select
+            value={tag}
+            onChange={(e) => {
+              setTag(e.target.value);
+            }}
+          >
+            <MenuItem value="food">food</MenuItem>
+            <MenuItem value="shelter">shelter</MenuItem>
+            <MenuItem value="health">health</MenuItem>
+            <MenuItem value="legal services">legal services</MenuItem>
+            <MenuItem value="shower">shower</MenuItem>
+            <MenuItem value="mission">mission</MenuItem>
+            <MenuItem value="social services">social services</MenuItem>
+          </Select>
+        </FormControl>
+      </Box>
+      <Box sx={{ display: 'flex' }}>
+        <Button
```

Comment:
there's probably a way to implement this with a map, (use array `[M, TU,W,TH,F,SA,SU]` and use both the elements and indices in the map), but it's up to you/whoever implements styling